CHANGELOG.md

```diff
@@ -12,13 +12,24 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+## v2.1.0 (WIP)
```

Comment:
nit: Think this is a patch update, and not a minor update.

No new features has been added.

Reply:
```suggestion
## v2.0.2 (WIP)
```

Reply:
```suggestion
## v2.0.2 (WIP)
```